src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -342,6 +343,27 @@ long MacroAssembler::get_const(address a) {
   }
   return (long) x;
 }
+// Branch-free implementation to convert !0 to false
+// Set register dst to true if dst is non zero using temp for calculations on Power Version<10.
+// Set register dst to true if dst is non zero for Power 10 and above machines.
+void MacroAssembler::normalize_bool(Register dst, Register temp, bool use_64bit) {
```

Comment:
I would've used `is_64bit/is_long` instead of `use_64bit`. But that's personal choice and leave it upto you.

Reply:
`is_64bit` would be fine with me, too. As you prefer.